setup.cfg

```diff
@@ -1,4 +1,62 @@
+[metadata]
+name = tiledb-ml
+version = 0.1.0
+description = Package supports all machine learning functionality for TileDB Embedded and TileDB Cloud
+author = TileDB, Inc.
+author_email = help@tiledb.io
+long_description = file: README.md
+long_description_content_type = text/markdown
+license = MIT
+license_file = LICENSE
+keywords = tiledb, ml
+url = https://github.com/TileDB-Inc/TileDB-ML
+platform = any
+project_urls =
+    Bug Tracker = https://github.com/TileDB-Inc/TileDB-ML/issues
+    Documentation = https://docs.tiledb.com
+classifiers =
+    Development Status :: 4 - Beta
+    Intended Audience :: Developers
+    Intended Audience :: Information Technology
+    Intended Audience :: Science/Research
+    Programming Language :: Python
+    Topic :: Software Development :: Libraries :: Python Modules
+    Topic :: Scientific/Engineering :: Artificial Intelligence
+    Operating System :: Unix
+    Operating System :: POSIX :: Linux
+    Operating System :: MacOS :: MacOS X
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    License :: OSI Approved :: MIT License
+
+[options]
+zip_safe = False
+packages = find_namespace:
+python_requires = >=3.6
+test_suite = tests
+install_requires = tiledb >= 0.9.0
+
+[options.extras_require]
+tensorflow = tensorflow>=2.5.0
+pytorch = torch>=1.8.1; torchvision>=0.9.1
+sklearn = scikit-learn>=0.23.0
+tensorflow_cloud = tensorflow>=2.5.0; tiledb-cloud>=0.7.11
+pytorch_cloud = torch>=1.8.1; torchvision>=0.9.1; tiledb-cloud>=0.7.11
+sklearn_cloud = scikit-learn>=0.23.0; tiledb-cloud>=0.7.11
+full = tensorflow>=2.5.0; torch>=1.8.1; torchvision>=0.9.1; scikit-learn>=0.23.0; tiledb-cloud>=0.7.11
```

Comment:
Have you tested this works as expected? I haven't seen the semicolon syntax before

Reply:
Found it [here](https://setuptools.readthedocs.io/en/latest/userguide/declarative_config.html?highlight=options.extras_require#opt-2). It says `list-semi`

Reply:
It should be ok imo. If you check in [TestPypi](https://test.pypi.org/project/tiledb-ml/) the package `0.2.0` is the result of this PR.